### PR TITLE
Sathishcg patch 1

### PR DESCRIPTION
**Pull request by sathishcg**
Updated the readme to add introduction about cv.

### CODE REVIEW

**Comment by j03wang** [comment]
I'd structure the README as follows:

1. **Correlation Vector for Java**: a blurb about the contents of this specific repository
2. **Correlation Vector**: a blurb about the cV and links to the specification and scenarios

An example cV is nice to see, but I don't think the full format spec is necessary here.

**Comment by sathishcg** [comment]
Thanks Joe for the feedback. 

#2 is covered. I will remove the background heading.

will add a blurb to cover #
1.
**regarding the spec** - as a 3rd party customer for adoption I will be really interested in the spec and format. Rolling up that upfront will be helpful rather than digging that through deep links. Thoughts?

**Comment by jacpull** [comment]
Remove the format description. The link to the spec repo is sufficient.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
 
+# Correlation Vector
+
+## Background
+
+**Correlation Vector** (a.k.a. **cV**) is a format and protocol standard for tracing and correlation of events through a distributed system based on a light weight vector clock.
+The standard is widely used internally at Microsoft for first party applications and services and supported across multiple logging libraries and platforms (Services, Clients - Native, Managed, Js, iOS, Android etc). The standard powers a variety of different data processing needs ranging from distributed tracing & debugging to system and business intelligence, in various business organizations.
+For more on correlation vector, read [future](https://github.com/Microsoft/CorrelationVector) roadmap and [scenarios](https://github.com/Microsoft/CorrelationVector/blob/master/Scenarios.md).
+
+This repo provides reference implementation for Java developers to take full advantage of cV in end-to-end distributed tracing scenarios.
```
Comment (jacpull) [inline README.md]: for Java developers to incorporate the cV for their distributed debugging and correlated analytics scenarios.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
 
+# Correlation Vector
+
+## Background
+
+**Correlation Vector** (a.k.a. **cV**) is a format and protocol standard for tracing and correlation of events through a distributed system based on a light weight vector clock.
+The standard is widely used internally at Microsoft for first party applications and services and supported across multiple logging libraries and platforms (Services, Clients - Native, Managed, Js, iOS, Android etc). The standard powers a variety of different data processing needs ranging from distributed tracing & debugging to system and business intelligence, in various business organizations.
+For more on correlation vector, read [future](https://github.com/Microsoft/CorrelationVector) roadmap and [scenarios](https://github.com/Microsoft/CorrelationVector/blob/master/Scenarios.md).
```
Comment (jacpull) [inline README.md]: For more on the correlation vector specification and the scenarios it supports, please see here

File: README.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+
+This repo provides reference implementation for Java developers to take full advantage of cV in end-to-end distributed tracing scenarios.
+
+# Format
```
Comment (jacpull) [inline README.md]: Suggest dropping this entirely.

**Comment by sathishcg** [comment]
@jacpull - Address the comments. Please review and approve.

File: README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 **Correlation Vector** (a.k.a. **cV**) is a format and protocol standard for tracing and correlation of events through a distributed system based on a light weight vector clock.
 The standard is widely used internally at Microsoft for first party applications and services and supported across multiple logging libraries and platforms (Services, Clients - Native, Managed, Js, iOS, Android etc). The standard powers a variety of different data processing needs ranging from distributed tracing & debugging to system and business intelligence, in various business organizations.
 
-For more on the correlation vector specification and the scenarios it supports, please refer the [specification](https://github.com/Microsoft/CorrelationVector)
+For more on the correlation vector specification and the scenarios it supports, please refer the [specification](https://github.com/Microsoft/CorrelationVector).
```
Comment (jacpull) [inline README.md]: to the

**Pull request merged**